edward/models/distributions.py

```diff
@@ -426,35 +491,30 @@ def __str__(self):
                "std dev: \n" + s.__str__()
 
     def sample_noise(self, size=1):
-        """
-        eps = sample_noise() ~ s(eps)
-        s.t. x = reparam(eps; params) ~ p(x | params)
-        """
         return tf.random_normal((size, ) + self.shape)
 
     def reparam(self, eps):
-        """
-        eps = sample_noise() ~ s(eps)
-        s.t. x = reparam(eps; params) ~ p(x | params)
-        """
         return self.loc + eps * self.scale
 
     def log_prob_idx(self, idx, xs):
-        """log p(xs[:, idx] | params[idx])"""
         full_idx = (slice(0, None), ) + idx # slice over batch size
         return norm.logpdf(xs[full_idx], self.loc[idx], self.scale[idx])
 
     def entropy(self):
         return tf.reduce_sum(norm.entropy(scale=self.scale))
 
 class PointMass(Distribution):
-    """
-    Point mass distribution
+    """Point mass distribution
 
-    p(x | params ) = prod_{idx in shape} Dirac(x[idx] | params[idx])
+    ``p(x | params ) = prod_{idx in shape} Dirac(x[idx] | params[idx])``
 
-    Dirac(x; p) is the Dirac delta distribution with density equal to
+    ``Dirac(x; p)`` is the Dirac delta distribution with density equal to
     1 if x == p and 0 otherwise.
+
+    Parameters
+    ----------
+    params : tf.Variable, optional
+        if not specified, everything initialized to :math:`\mathcal{N}(0,1)`
```

Comment:
the parameters for `PointMass` are not described anywhere else, as far as i know.

i linked all the other distributions to their counterparts in `edward.stats.distributions`. (again, i think this is rather confusing.)

i added `np.ndarray` but kept the block.